content/blog/2021/08/2021-08-14-introducing-junit-external-storage.adoc

```diff
@@ -0,0 +1,147 @@
+---
+:layout: post
+:title: "Introducing external storage for JUnit test results"
+:tags:
+- cloud-native
+- pluggable-storage
+- junit
+:author: timja
+:sig: gsoc
+// opengraph: // TODO
+//   image:
+links:
+//   discourse: TODO
+
+---
+
+In common CI/CD use-cases a lot of the space is consumed by test reports.
+This data is stored within `JENKINS_HOME`,
+and the current storage format requires huge overheads when retrieving statistics and, especially, trends.
+In order to display trends, each report has to be loaded and then processed in-memory.
+
+The main purpose of externalising Test Results is to optimize Jenkins performance and storage
+by querying the desired data from external storages.
+
+I'm please to announce that the plugin:junit[JUnit Plugin] external storage is now available for use.
+
+== Getting started
+
+Install your database vendor specific plugin, you can use the Jenkins plugin site to search for it:
+
+https://plugins.jenkins.io/ui/search/?labels=database
```

Comment:
Beware that https://plugins.jenkins.io/database/#dependencies shows this to be incomplete: https://plugins.jenkins.io/database-drizzle/

Reply:
It's missing a label to show up, but not sure if it should be encouraged, it has no docs and hasn't been released in 9 years